### PR TITLE
Add file for timIdeaTech Edmonton

### LINKED ISSUE #42: Create a new file for timIdeaTech Edmonton
- [ ] Create a branch
- [ ] Add commits
- [ ] Open a PR
- [ ] Discuss
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by timIdeaTech**
closes #42

### CODE REVIEW

File: timIdeaTech-Edmonton.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Edmonton, AB Canada
+
+### Recomendations
+- Eat at Siagon Cuisine
+
+### Fun Facts
+- It's snowing (go figure)
```
Comment (cmr1121) [inline timIdeaTech-Edmonton.md:7]: Under a foot is just a flurry... :rofl:

**Comment by cmr1121** [comment]
Love Edmonton!

**Review by cmr1121 — APPROVED** [review]
:us:  vs :canada: today in world hockey juniors.  Good luck!

**Comment by timIdeaTech** [comment]
Should be good!

**Pull request merged**